images/win/scripts/Installers/Wait-WindowsUpdatesForInstall.ps1

```diff
@@ -0,0 +1,12 @@
+################################################################################
+##  File:  Wait-WindowsUpdatesForInstall.ps1
+##  Desc:  Wait for installation windows updates to complete
+################################################################################
+
+Invoke-SBWithRetry -RetryCount 10 -TimeoutInSecs 120 -Command {
```

Comment:
I guess, "TimeoutInSecs" can confuse since actually it's Sleep time in secs. Could you please correct parameter naming? Perhaps for some long term operations we'll need to add actual timeout in future.

Reply:
I don't see any difference between Sleep vs Timeout in particular case.

```
PS >timeout /?

Description:
    This utility accepts a timeout parameter to wait for the specified
    time period (in seconds) or until any key is pressed. It also
    accepts a parameter to ignore the key press.

Parameter List:
    /T        timeout       Specifies the number of seconds to wait.
```

Reply:
Timeout operation terminates process after specified time (if it was not completed yet) on the other hand Sleep operation just waits for specified time and proceeds execution after that.